CollectionvViewDemo/Cells/DailyEntryCollectionViewCell.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  DailyEntryCollectionViewCell.swift
+//  CollectionvViewDemo
+//
+//  Created by Bence Pattogato on 23/08/2021.
+//
+
+import UIKit
+
+class DailyEntryCollectionViewCell: UICollectionViewCell {
+    
+    @IBOutlet weak var dayLabel: UILabel!
+    @IBOutlet weak var iconLabel: UILabel!
+    @IBOutlet weak var minTempLabel: UILabel!
+    @IBOutlet weak var maxTempLabel: UILabel!
```

Comment:
[nit] `private` and strong? I know it's just a showcase app so it's not that important, but I think it would worth to follow our conventions to mirror the quality of our everyday work, especially if it's not that big of a hassle.